test/instrumentation/testdata/stable-metrics-list.yaml

```diff
@@ -141,40 +141,24 @@
   buckets:
   - 0.05
   - 0.1
-  - 0.15
```

Comment:
Probably you want to revert these changes ;)

Reply:
Why? Actually - those are needed as we're changing the buckets for the metric.